graphql-dgs-codegen-core/src/test/resources/kotlin2/constantsForInputTypes/schema.graphql

```diff
@@ -0,0 +1,12 @@
+type Query {
```

Comment:
For example, this schema will generate the kotlin classes under `test/resources/kotlin2/constantsForInputTypes/expected/...`